### ISSUE TITLE
Managed to Google SignIn. Now want to GET gmail. 

### ISSUE THREAD

**Issue by petercutting** [EddyVerbruggen/nativescript-plugin-firebase#509]
Hi
I have managed to Google signin (android). (So now I have a UID amongst other things)

How do I GET gmails using REST? I suspect I need some kind of gmail api token which I supply in the GET. If so how do I get the token?

Also is there a nativescript library that I can use to get the gmails or is it just plain HTTP.

Thanks
Peter

**Comment by richarddavenport** [comment]
I don't think this is the library you're looking for. This is for firebase. I'm very familiar with firebase, but not many other Google services. The last time I looked you had to have a gapi token, and that is _not_ the same as a firebase user token.
